packages/manager/src/features/ObjectStorage/AccessKeyLanding/HostNamesDrawer.tsx

```diff
@@ -30,9 +31,12 @@ export const HostNamesDrawer = (props: Props) => {
         <CopyAllHostnames
           text={
             regions
-              .map(
-                (region) =>
-                  `${regionsLookup[region.id]?.label}: ${region.s3_endpoint}`
+              .map((region) =>
+                region?.endpoint_type
+                  ? `${regionsLookup[region.id]?.label} (${
+                      region.endpoint_type
+                    }): ${region.s3_endpoint}`
+                  : `${regionsLookup[region.id]?.label}: ${region.s3_endpoint}`
               )
               .join('\n') ?? ''
```

Comment:
Slightly more readable
```suggestion
              .map((region) => {
                const label = regionsLookup[region.id]?.label;
                const endpointType = region.endpoint_type
                  ? ` (${region.endpoint_type})`
                  : '';
                return `${label}${endpointType}: ${region.s3_endpoint}`;
              })
              .join('\n') ?? ''
```